galasa-ui/src/utils/3270/appendMetadataStatusLine.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+import { TerminalImageCharacter } from '@/utils/interfaces/3270Terminal';
+
+export function appendMetadataStatusLine(
+  columns: number,
+  rows: number,
+  characterArray: (TerminalImageCharacter | null)[][],
+  imageId: string,
+  inbound: boolean | undefined,
+  aid: string | undefined
+) {
+  let inboundOrOutboundText = '',
+    aidKeyText = '';
+
+  inboundOrOutboundText = inbound ? ' - Inbound' : ' - Outbound';
```

Comment:
Very minor suggestion - can we rename the `inbound` parameter to `isInbound` to make it 100% obvious that it's a boolean wherever it's used within this function?